### DISCUSSION TITLE
Digest Analog from Proscan

### DISCUSSION THREAD

**Discussion by bobfeser** [chuot/rdio-scanner#350] [Q&A]
I have 2 frequencies that are analog that are recorded from Proscan that will not ingest. Any ideas? I am new to RDIO Scanner and trying to get it set up.  I have Bell Fleetnet already figured out except for some little tweaks her and there. 

https://www.radioreference.com/db/sid/2560

The following is the information.

FIre:

![image](https://user-images.githubusercontent.com/123333205/236874436-667c4def-d20d-4594-b69b-c9c3ed5459c6.png)

OPP Repeater:

![image](https://user-images.githubusercontent.com/123333205/236874753-bde60a9f-99bb-4711-80eb-81d38c6c58cf.png)

I do realize that Fire and OPP are NOT part of the Fleetnet system.

Thanks,

Bob

**Comment by bobfeser** [comment]
this is what it is telling me,

![image](https://user-images.githubusercontent.com/123333205/236876326-87078a9d-f01a-4cd2-8ac7-d281fc115289.png)

**Reply by noamlivne** [reply]
Indeed, Proscan doesn't show a TalkGroup in the example you pasted. 
You need to create a Proscan mask via the Options > Recorder tab that contains a TalkGroup option.
Or create a fictional TalkGroup that will CatchAll the recordings and assign them with the same TalkGroup, via the Rdioscanner Dirwatch tab.
